informer/metric.go

```diff
@@ -10,6 +10,15 @@ const (
 )
 
 var (
+	cacheLastUpdatedGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: PrometheusNamespace,
+			Subsystem: PrometheusSubsystem,
+			Name:      "cache_last_updated",
+			Help:      "A gauge metric expressing the time of the last cache update.",
+		},
+	)
```

Comment:
Did you consider using counter for that? Then we can check if `reate(...[1h]) == 0` in the alert. With counter we have a little bit more potentially useful information.

Reply:
With that approach it works both I guess. Where would be the more useful information with a counter? We want to do exactly one thing.

Reply:
See https://www.robustperception.io/are-increasing-timestamps-counters-or-gauges

> So timestamps should always be gauges, and generally reported using Unix timestamps in seconds.

I will 100% fallback to Brian's input on this one :D

Reply:
Pawel's suggestion is to not use a timestamp, but a counter. I think both would work but I would just go with a timestamp because I do not see the added value or the better approach in the counter version.

Reply:
I think both have value.
With a counter, we can get rates a lot easier. With timestamp, we can tell when the last loop was. I don't see why we can't/shouldn't add both.